src/Lykke.AzureStorage/Tables/Paging/PagedResult.cs

```diff
@@ -3,20 +3,26 @@
 
 namespace Lykke.AzureStorage.Tables.Paging
 {
-    public class PagedResult<T>: IEnumerable<T>
+    public interface IPagedResult<out T> : IEnumerable<T>
     {
-        public IEnumerable<T> Result { get; }
+        PagingInfo PagingInfo { get; }
+    }
+
+    public class PagedResult<T>: IPagedResult<T>
+    {
+        private readonly IEnumerable<T> _result;
+
         public PagingInfo PagingInfo { get; }
 
         public PagedResult(IEnumerable<T> items = null, PagingInfo pagingInfo = null)
```

Comment:
```items = null``` may lead to NullReferenceException in ```GetEnumerator()```, so it's bad default value.

Reply:
adde initialisation in case parameter is null